runtime/dolphin/src/tests/mod.rs

```diff
@@ -1036,7 +1036,7 @@ fn send_para_a_asset_to_para_b_with_trader_and_fee() {
 	ParaA::execute_with(|| {
 		assert_ok!(AssetManager::register_asset(
 			parachain::Origin::root(),
-			// This need to be changed starting from v0.9.16
+			// This need to be changed starting from v0.9.18
```

Comment:
I'd just get rid of this comment. We have an issue logged for exploring this change.